migrations/20190128134129-add-room-origin-column.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
@iamigo I'm just curious but will this migration run if this is deployed now even though the date time string is back from January?

Reply:
I *think* so, because sequelize maintains a list of migrations which it has executed in the SequelizeMeta table, no timestamps in that table, just the migration file names.

BUT it would be great to vet this in staging or devbox